nginx/datadog_checks/nginx/nginx.py

```diff
@@ -148,9 +153,10 @@ def _get_instance_params(cls, instance):
         url = instance.get('nginx_status_url')
 
         use_plus_api = instance.get("use_plus_api", False)
+        use_plus_api_stream = instance.get("use_plus_api_stream", False)
```

Comment:
Default value needs to be `True` to keep compatibility with other versions, if not the plus_api_stream would get disabled for other users when updating to newer agent versions

Reply:
Done